mpa/modules/xai/builder.py

```diff
@@ -0,0 +1,2 @@
+def build_explainer(model, cfg):
```

Comment:
It seems that this function is not called and has no meaning, if I am right, how about remove?

Reply:
This remained for the initial design suggestion but can be removed, and added when future implemented in the future.